### PR TITLE
Make take interview link open in a separate tab

### LINKED ISSUE #816: Video Interview: open Jobma link in a new tab
As a user, I lose my place in the application process when I go to the Jobma site, and there's now way for me to link back. 

It would be convenient if Jobma opened in a new tab.

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #816 

#### What's this PR do?
Removes the view to create a Jobma interview and the frontend logic which uses it. Instead, the interview will be created via an API call in the backend, with a celery task, when the `BootcampApplication` is first created

#### How should this be manually tested?
First, make sure the relevant models are created:
 - `Job` with the right template id linking to a bootcamp run you want to test with
 -  `ApplicationStep` for the bootcamp with type  "Video Interview"
 -  `BootcampRunApplicationStep` linking to it and a bootcamp run

Create a new `BootcampApplication` for that run (or try deleting and reenrolling via the UI). The application should start a celery task which populates the `Interview` object, and `Interview.interview_url`, for the application.

On the dashboard, click the link to open the video interview tab. You should see a "Take Video Interview" link. Click it and you should have a new tab or window open with a link to the Jobma interview to be started.

### CODE REVIEW

File: applications/api.py [hunk 1]
```diff
@@ -89,3 +102,34 @@ def get_required_submission_type(application):
...
+
+
+def populate_interviews_in_jobma(application):
+    """
+    Go over each ApplicationStep for the application and create the interviews in Jobma.
+
+    Args:
+        application (BootcampApplication): A bootcamp application
+    """
+    for run_step in BootcampRunApplicationStep.objects.filter(
```
Comment (gsidebo) [inline applications/api.py:114]: We need to filter this to only include run steps with `application_step__submission_type=SUBMISSION_VIDEO`

File: applications/serializers.py [hunk 2]
```diff
@@ -41,14 +41,24 @@ class SubmissionSerializer(serializers.ModelSerializer):
     """ApplicationStepSubmission serializer"""
 
     interview_url = serializers.SerializerMethodField(required=False, allow_null=True)
+    take_interview_url = serializers.SerializerMethodField(
+        required=False, allow_null=True
+    )
 
     def get_interview_url(self, submission):
-        """ Get the interview url for a video submission if of that type"""
+        """Return the results URL for the reviewer or others to view the interview"""
```
Comment (gsidebo) [inline applications/serializers.py:49]: Can you rename this to `get_result_interview_url` to disambiguate more clearly?

**Comment by noisecapella** [comment]
@gsidebo `submission_status` matches `review_status` which is also non-nullable. `Order.status` and `BootcampApplication.state` are also non-nullable. Is there a meaningful difference between having `ApplicationStepSubmission.status == PENDING` and `ApplicationStepSubmission.status is None`?

File: applications/api.py (hunk 1 above)
Comment (noisecapella) [reply applications/api.py:114]: Thanks, good catch

**Comment by gsidebo** [comment]
@noisecapella I guess that works. Would be nice to have a status that differentiates submissions that have not been started, and ones that have but cannot be completed. We can make that change later if we need to I guess

File: applications/serializers.py (hunk 2 above)
Comment (noisecapella) [reply applications/serializers.py:49]: I agree this is confusing but I'd like to address this later in the interest of time

**Comment by noisecapella** [comment]
@gsidebo any other comments? @pdpinch created a new item #831 which depends on this so we should merge soon if everything looks good

**Comment by gsidebo** [comment]
@noisecapella Testing now

**Comment by noisecapella** [comment]
@gsidebo looks like I need to tweak the CSS. The link is turning white on hover.

**Pull request merged**